js-loops_for-console/index.js

```diff
@@ -1 +1,4 @@
 console.clear();
+for (let index = 0; index <= 100; index++) {
+  console.log(index);
```

Comment:
Put a tag/ text before your console log so you know what you are console logging.